Setup.sh

```diff
@@ -3,36 +3,27 @@
 # EXPERIMENTAL!
 # This file is experimental and is not maintained directly by Improbable. Please use at your own risk.
 
-set -e -u -o pipefail
-if [ -n "${TEAMCITY_CAPTURE_ENV:-}" ]; then set -x; else set +x; fi
+set -e -u -o -x pipefail
 
 if [ "$(uname -s)" != "Darwin" ]; then
     echo "This script should only be used on OS X. If you are using Windows, please run Setup.bat."
     exit 1
 fi
 
 function markStartOfBlock {
-    if [ -n "${TEAMCITY_CAPTURE_ENV:-}" ]; then
-        echo -e "/x23/x23teamcity[blockOpened name='$1']"
-    else
-        echo "Starting: $1"
-    fi
+    echo "Starting: $1"
```

Comment:
Question I possibly should already know the answer to. What are we expecting this argument to be? We don't seem to check for it's existence and I thought we were supposed to run this script with no arguments.

Reply:
Ah never mind. Misread.